app/scripts/controllers/membership.js

```diff
@@ -88,16 +91,16 @@ angular
           .then(function() {
             resetForm();
             refreshRoleBindingList();
-            showAlert('rolebindingCreate', 'success', messages.update.subject.success({
+            showAlert('rolebindingCreate', 'success', escaped.update.subject.success({
```

Comment:
Ok I can update this.  Something about it still bothers me, looking at the docs:

`ngBindHTML`:

> Evaluates the expression and inserts the resulting HTML into the element in a secure way. By default, the resulting HTML content will be sanitized using the $sanitize service. 

`$sanitize`:

> Sanitizes an html string by stripping all potentially dangerous tokens.